packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -216,9 +202,8 @@ const Injector = ({ children }: { children: React.ReactNode }): JSX.Element => {
     const network = isConnectedToArbitrum ? l2Network : l1Network
     const networkId = String(network.chainID)
 
-    // TODO: Deprecate network utils in favor of @arbitrum/sdk networks
     actions.app.reset(networkId)
-    actions.app.setNetworkID(networkId)
+    actions.app.setNetworks({ l1Network, l2Network })
```

Comment:
This is temporary, we'll get rid of networks from the global state in favor of the `useNetworks` hook, eventually.